db/schema.yml

```diff
@@ -1232,6 +1232,7 @@ generic_objects:
 - generic_object_definition_id
 - created_at
 - updated_at
+- properties
```

Comment:
I don't truly understand either. I guess it has to do with the new migration requirement. @carbonin can you confirm?